lib/mayaUsd/ufe/ProxyShapeHierarchy.cpp

```diff
@@ -257,11 +253,11 @@ Ufe::UndoableCommand::Ptr ProxyShapeHierarchy::reorderCmd(const Ufe::SceneItemLi
 	    orderedTokens.emplace_back(downcast(item)->prim().GetPath().GetNameToken());
 	}
 
-	// TODO HS Oct 23, 2020, grab any child and pass it to UsdUndoReorderCommand
-	// in order to get the parent out of it.
+	// create a reorder command and pass in the parent and its ordered children list
     const auto& childPrim = downcast((*orderedList.begin()))->prim();
+    const auto& parentPrim = childPrim.GetParent();
 
-	return UsdUndoReorderCommand::create(childPrim, orderedTokens);
+	return UsdUndoReorderCommand::create(parentPrim, orderedTokens);
```

Comment:
Of course there's nothing wrong with this, but you can avoid the downcast and the call to childPrim.GetParent(), because here you know the parent prim is the pseudo-root, so you can just pass it:

return UsdUndoReorderCommand::create(getUsdRootPrim(), orderedTokens);

Reply:
@ppt-adsk addressed in 520b36de46f29948c6fc3ad99a67f2080c3f1cda